boards: st: nucleo_u575zi_q: add flash partition

whati001 opened the pull request:
Added flash partition to board nucleo_u575zi_q, enabling flash and mcuboot sample

etienne-lms commented on boards/st/nucleo_u575zi_q/nucleo_u575zi_q.dts:
```diff
@@ -34,6 +35,44 @@
+			label = "storage";
+			reg = <0x001f0000 DT_SIZE_K(64)>;
+		};
+
```
Could you remove this empty line?

whati001 replied:
Updated

etienne-lms commented on boards/st/nucleo_u575zi_q/nucleo_u575zi_q.dts:
```diff
@@ -34,6 +35,44 @@
+			label = "mcuboot";
+			reg = <0x00000000 DT_SIZE_K(64)>;
+		};
+		slot0_partition: partition@10000 {
+			label = "image-0";
+			reg = <0x00010000 DT_SIZE_K(928)>;
+		};
```
Could you add an empty line before and after this node.

etienne-lms commented on boards/st/nucleo_u575zi_q/nucleo_u575zi_q.dts:
```diff
@@ -34,6 +35,44 @@
+			reg = <0x000f8000 DT_SIZE_K(928)>;
+		};
+
+		scratch_partition: partition@1e0000 {
```
It seems scratch partition is no more used by MCUBoot and could be removed.

whati001 replied:
Updated

etienne-lms reviewed (commented):
There should be a message body in the commit message, e.g.:
```
boards: st: nucleo_u575zi_q: add flash partition

Add flash partition description in nucleo_u575zi_q board DTS
file to ease integration of MCUBoot as bootloader stage.

Signed-off-by: ...
```
*(edited: note that CI Compliance Check also complains on that)*

Commit:
boards: st: nucleo_u575zi_q: add flash partition

Add flash partition description in nucleo_u575zi_q board DTS
file to ease integration of MCUBoot as bootloader stage.

Signed-off-by: Andreas Karner <andreas.karner@sloc.one>
